packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -503,6 +538,8 @@ const Dropdown = ({
                         portal_id={list_portal_id}
                         ref={list_ref}
                         suffix_icon={suffix_icon}
+                        should_scroll_to_selected={should_scroll_to_selected}
+                        should_autohide={should_autohide}
```

Comment:
```suggestion
                        should_autohide_scrollbars={should_autohide_scrollbars}
```
just to clarify what needs to be autohidden :)

Reply:
autohide is true by default. I'm making it false for my component in order not to show that white space

Reply:
i think maryia is asking you to rename the variable to make it more clear